### DISCUSSION TITLE
请问UMI能直接输出ES6的JS给浏览器用么

### DISCUSSION THREAD

**Discussion by Aloento** [umijs/umi#7387] [Q&A] [answered]
我是个新手，然后有一个很傻的问题：
请问UMI能直接输出**ES6**甚至更新的语言版本的JS给浏览器用么，不考虑 **旧** 浏览器兼容性
我现在在用TS写程序，然后试了试 **#** 标识符，发现生成出来的是名为 **__private_0_** 的对象

上网搜了搜没找到太多的相关资料，遂来此询问，求大佬赐教

又去搜了搜，发现了 **babel-preset-env** 这个东西，UMI有用它么，我没找到在哪里修改UMI的Babel配置

**Comment by 010101010jasgjsgdga** [comment, marked as answer]
UMI封装了webpack  不直接暴露相关配置，具体配置查手册吧，配置了也不一定好使
